test/coverlet.collector.tests/CoverletSettingsParserTests.cs

```diff
@@ -72,6 +72,41 @@ public void ParseShouldCorrectlyParseConfigurationElement()
             Assert.True(coverletSettings.SingleHit);
         }
 
+        [Theory]
+        [InlineData(" , json", 2, new []{ " ", " json" })]
+        [InlineData(" , json, ", 3, new [] { " ", " json", " " })]
+        [InlineData("json,cobertura", 2, new[] { "json", "cobertura" })]
+        [InlineData(" , json,, cobertura ", 3, new[] { " ", " json", " cobertura " })]
+        [InlineData(" , json, , cobertura ", 4, new [] { " ", " json", " ", " cobertura " })]
```

Comment:
mmm we should remove empty entries...so this should be 2 `json,cobertura` isn't it?
And also we need to trim format after parsing to normalize to `cobertura`(len 9) and not ` cobertura `(len 11)

Reply:
The current format parser only checks for ```null``` and ```string.Empty``` so I also implemented it this way. [https://github.com/tonerdo/coverlet/blob/b4701d260b65a35d36185129f67bd14316bca631/src/coverlet.collector/DataCollection/CoverletSettingsParser.cs#L90](https://github.com/tonerdo/coverlet/blob/b4701d260b65a35d36185129f67bd14316bca631/src/coverlet.collector/DataCollection/CoverletSettingsParser.cs#L90)

I agree with you and think it would make sense to trim the strings at this point. Then also 'whitespace' only strings would be ignored. 
I changed it now.

Reply:
Done